### DISCUSSION TITLE
Is it possible to do ''POST', '/session/${sessionId}/${vendor_prefix}/cdp/execute'?

### DISCUSSION THREAD

**Discussion by VicGrygorchyk** [webdriverio/webdriverio#6618] [Q&A] [answered]
Hello
Selenium project has such command:
```
'POST', '/session/${sessionId}/${vendor_prefix}/cdp/execute'
```
I am especially interested in doing via WDIO next:
```
// Selenium code
var cmd = 'Network.emulateNetworkConditions'
var args = {offline: true, latency: 250, downloadThroughput: 1, uploadThroughput: 1, connectionType: 'none'}
driver.execute("executeCdpCommand", {cmd: cmd, params: args})
```
Would you, please, point me out is the same could be achieved via WDIO?
Thank you for your great instrument!
Regards,

**Comment by BorisOsipov** [comment, marked as answer]
@VicGrygorchyk can [this ](https://webdriver.io/docs/customcommands#add-more-webdriver-commands) solve your issue?

**Reply by VicGrygorchyk** [reply]
Yes, it seems like a good workaround.

**Reply by VicGrygorchyk** [reply]
I've also found https://webdriver.io/docs/api/chromium/#setnetworkconnection, it seems like what I was looking for

**Comment by christian-bromann** [comment]
@VicGrygorchyk can you point me to the new Selenium 4 commands so I can add them to the protocol?
